sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeedAccessMode.cs

```diff
@@ -5,17 +5,19 @@
 
 namespace Azure.AI.MetricsAdvisor.Models
 {
-    // TODODOCS.
     /// <summary>
+    /// The view mode of the <see cref="DataFeed"/>.
     /// </summary>
     [CodeGenModel("ViewMode")]
     public readonly partial struct DataFeedAccessMode
     {
         /// <summary>
+        /// Indicates that the data feed is private.
```

Comment:
This could be me missing content, but private to whom?

Reply:
only admins can see it?

Reply:
We need to clarify this.